### PR TITLE
Added readme in Thai 

### PR DESCRIPTION
**Pull request by grimreapermanasvi**
READ ME in Thai language

# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by ArthurZucker** [comment]
cc @stevhliu

File: README_th.md [hunk 1]
```diff
@@ -0,0 +1,534 @@
...
+
+<h4 align="center">
+    <p>
+        <a href="https://github.com/huggingface/transformers/">English</a> |
+        <a href="https://github.com/huggingface/transformers/blob/main/README_zh-hans.md">简体中文</a> |
+        <a href="https://github.com/huggingface/transformers/blob/main/README_zh-hant.md">繁體中文</a> |
+        <a href="https://github.com/huggingface/transformers/blob/main/README_ko.md">한국어</a> |
+        <b>Thai</b> |
+        <a href="https://github.com/huggingface/transformers/blob/main/README_ja.md">日本語</a> |
+        <a href="https://github.com/huggingface/transformers/blob/main/README_hd.md">हिन्दी</a>
```
Comment (stevhliu) [inline README_th.md:49]: Lets also add links to the Russian and Portuguese translations here.

File: README_th.md [hunk 2]
```diff
@@ -0,0 +1,534 @@
...
+- [การแปลโดยใช้ T5](https://huggingface.co/t5-base?text=My+name+is+Wolfgang+and+I+live+in+Berlin)
+
+ในการมองเห็นของคอมพิวเตอร์:
```
Comment (stevhliu) [inline README_th.md:90]: Missing links to MaskFormer, DPT, and VideoMAE in this section.

File: README_th.md [hunk 3]
```diff
@@ -0,0 +1,534 @@
...
+- [การจัดหมวดหมู่รูปภาพด้วย ViT](https://huggingface.co/google/vit-base-patch16-224)
+- [การตรวจจับวัตถุด้วย DETR](https://huggingface.co/facebook/detr-resnet-50)
+- [Segmentación semánticaด้วยSegFormer](https://huggingface.co/nvidia/segformer-b0-finetuned-ade-512-512)
+- [Segmentación panópticaด้วยDETR](https://huggingface.co/facebook/detr-resnet-50-panoptic)
```
Comment (stevhliu) [inline README_th.md:94]:
```suggestion
```

File: README_th.md [hunk 4]
```diff
@@ -0,0 +1,534 @@
...
+- [Segmentación Universalด้วยOneFormer (Segmentación Semántica,ของ Instance และ Panoptic ด้วยโมเดลเดียว)](https://huggingface.co/shi-labs/oneformer_ade20k_dinat_large)
+
+ในส่วนของเสียง:
```
Comment (stevhliu) [inline README_th.md:97]: Missing the Audio Spectrogram Transformer here

File: README_th.md [hunk 5]
```diff
@@ -0,0 +1,534 @@
...
+- [การรับรู้เสียงอัตโนมัติด้วย Wav2Vec2](https://huggingface.co/facebook/wav2vec2-base-960h)
+- [การตรวจจับคำสำคัญด้วย Wav2Vec2](https://huggingface.co/superb/wav2vec2-base-superb-ks)
+
+ในงานที่รวมแบบหลายสื่อ:
```
Comment (stevhliu) [inline README_th.md:101]: Missing TAPAS, CLIP, LayoutLM, and X-CLIP here

File: README_th.md [hunk 6]
```diff
@@ -0,0 +1,534 @@
...
+- [การตอบคำถามทางสายตาด้วย ViLT](https://huggingface.co/dandelin/vilt-b32-finetuned-vqa)
+
+**[เขียนด้วย Transformer](https://transformer.huggingface.co)**, ที่สร้างโดยทีม Hugging Face เป็นการสาธิตอย่างเป็นทางการของความสามารถในการสร้างข้อความในคลังข้อมูลนี้
+
```
Comment (stevhliu) [inline README_th.md:105]: Missing section about the 100 projects using Transformers here

File: README_th.md [hunk 7]
```diff
@@ -0,0 +1,534 @@
...
+1. ปรับแต่งโมเดลหรือตัวอย่างตามความต้องการได้อย่างง่าย:
+- เราให้ตัวอย่างสำหรับแต่ละโครงสร้างเพื่อทำการทดลองให้ได้ผลลัพธ์เท่าเดิมกับผลลัพธ์ที่เผยแพร่โดยผู้แต่งเรื่องต้น.
+- ส่วนในโมเดลถูกเปิดเผยออกมาให้เห็นที่สุดเท่าที่จะเป็นไปได้.
+- แฟ้มโมเดลสามารถใช้งานได้อย่างอิสระจากไลบรารีสำหรับการทดลองที่รวดเร็ว
+
+## ทำไมคุณไม่ควรใช้ Transformers?
+
+- ห้องสมุดนี้ไม่ใช่ชุดเครื่องมือที่สามารถปรับแต่งแบบบล็อกสำหรับเครือข่ายประสาทเสมือน รหัสในไฟล์ของโมเดลไม่ได้รับการรีแฟคทอรี่ด้วยการสรุปออกเสมือนเพิ่มขึ้นอนุมานแต่จะให้นักวิจัยสามารถทำการวนเร็วในแต่ละโมเดลโดยไม่ต้องตกหลุมลงในสรุป/ไฟล์เพิ่มเติม
+-API การฝึกไม่ถูกออกแบบมาเพื่อทำงานกับโมเดลใดๆ แต่ถูกปรับแต่งให้ทำงานได้ดีกับโมเดลที่ไลบรารีนี้ให้ สำหรับวงรอบของการเรียนรู้เครื่องจักรทั่วไป คุณควรใช้ไลบรารีอื่น (อาจจะ [Accelerate](https://huggingface.co/docs/accelerate)).
```
Comment (stevhliu) [inline README_th.md:220]:
```suggestion
- API การฝึกไม่ถูกออกแบบมาเพื่อทำงานกับโมเดลใดๆ แต่ถูกปรับแต่งให้ทำงานได้ดีกับโมเดลที่ไลบรารีนี้ให้ สำหรับวงรอบของการเรียนรู้เครื่องจักรทั่วไป คุณควรใช้ไลบรารีอื่น (อาจจะ [Accelerate](https://huggingface.co/docs/accelerate)).
```

**Review by stevhliu — COMMENTED** [review]
Thanks for starting the Thai translations. There are several missing links to models as well as a section :)

File: README_th.md (hunk 1 above)
Comment (grimreapermanasvi) [reply README_th.md:49]: sure will work on it

**Pull request closed without merging**